Azure/sonic-buildimage
# Add thermal plan to as4630_54pe

**- What I did**
Add thermal plan to as4630_54pe
**- How I did it**
Add thermal plan to as4630_54pe
**- How to verify it**
Remove fan and accton_as4630_54pe_monitor.py  to check that temp change. And Fan will change depend od thermal policy.
root        : DEBUG    ori_duty_cycle=50
root        : DEBUG    test_temp=1
root        : DEBUG    temp[0]=52000, temp[1]=42000, temp[2]=52000
root        : DEBUG    temp_val=146000, fan_fail=0
root        : DEBUG    temp=146000 > fan_policy[0][2]=0
root        : DEBUG    temp=146000 > fan_policy[1][2]=140000
root        : DEBUG    temp=146000 <= fan_thermal_policy[1][3]=150000
root        : DEBUG    new_duty_cycle=62
root        : DEBUG    ori_state=0, current_state=1, temp_val=146000


root        : DEBUG    ori_duty_cycle=56
root        : DEBUG    test_temp=1
root        : DEBUG    temp[0]=54000, temp[1]=44000, temp[2]=54000
root        : DEBUG    temp_val=152000, fan_fail=0
root        : DEBUG    temp=152000 > fan_policy[0][2]=0
root        : DEBUG    temp=152000 > fan_policy[1][2]=140000
root        : DEBUG    temp=152000 > fan_policy[2][2]=150000
root        : DEBUG    temp=152000 <= fan_thermal_policy[2][3]=160000
root        : DEBUG    new_duty_cycle=75
**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
